src/components/content/catalog/services/ServiceProvider.tsx

```diff
@@ -109,12 +111,44 @@ function ServiceProvider({
         setActiveKey(key);
     };
 
+    function setUnregisterTipsInfo(type: 'error' | 'success', msg: string) {
+        setUnregisterTips(
+            <div className={'submit-alert-tip'}>
+                {' '}
+                <Alert
+                    message='Unregister:'
+                    description={msg}
+                    showIcon
+                    type={type}
+                    closable={true}
+                    onClose={onRemove}
+                />{' '}
+            </div>
+        );
+    }
+
+    const onRemove = () => {
+        window.location.reload();
+    };
+
+    const [unregisterTips, setUnregisterTips] = useState<JSX.Element | undefined>(undefined);
+
+    const onConfirmUnregister = useCallback((type: string, msg: string) => {
+        setUnregisterTipsInfo(type as 'success' | 'error', msg);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, []);
+
     return (
         <>
             {serviceName.length > 0 ? (
                 <>
+                    {unregisterTips}
                     <Tabs items={items} onChange={onChange} activeKey={activeKey} className={'ant-tabs-tab-btn'} />
                     <ServiceDetail serviceDetails={serviceDetails} serviceAreas={serviceAreas} />
+                    <div className={'update-unregister-btn-class'}>
+                        <UpdateService id={serviceDetails.id} />
+                        <UnregisterService id={serviceDetails.id} onConfirmHandler={onConfirmUnregister} />
```

Comment:
when the unregister is successful, we must not display the update and register buttons. We can either disable it or not display at all.

Reply:
fixed.